File: codeforces/13.bear_and_big_brother_[791A]/babb.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int limak_weight, bob_weight;
  unsigned short int years = 0;
  cin >> limak_weight >> bob_weight;
  while (limak_weight <= bob_weight) {
    limak_weight = limak_weight * 3;
    bob_weight = bob_weight * 2;
    years++;
  }
  cout << years << endl;
  return 0;
}

File: codeforces/low_difficulty/difficulty_800/13.bear_and_big_brother_[791A]/babb.py
# Author: José Rodolfo (jric2002)
weights = input().split(" ")
limak_weight = int(weights[0])
bob_weight = int(weights[1])
years = 0
while (limak_weight <= bob_weight):
  limak_weight = limak_weight * 3
  bob_weight = bob_weight * 2
  years += 1
print(years)